content/sensu-go/6.4/operations/deploy-sensu/scale-event-storage.md

```diff
@@ -435,46 +421,38 @@ After you enter your password, Postgres will list database copy progress:
 30318/30318 kB (100%), 1/1 tablespace
 {{< /code >}}
 
-### Step 4: Confirm replication
-
-To confirm your configuration is working properly, first remove configurations that are only for the primary:
+### Confirm replication
 
+1. On the **primary** Postgres host, remove primary-only configurations:
```

Comment:
Need to confirm that this step should indeed be completed on the primary host. Does it matter which host users run this command from?